source/libSteve/opengl2.c

```diff
@@ -400,6 +401,12 @@ void gl2glutInit(int *argc,char **argv) {
 		glutInit(&defaultc,(char**) defaultv);
 		free(defaultv[0]);
 		free(defaultv); }
+
```

Comment:
glut is tricky to get right on OSX. Reverted all glut related changes in this PR. I'll deal with them later in a separate PR.